File: commons/javafx/src/test/java/com/cedarsoft/commons/javafx/properties/KotlinFxPropertiesJavaTest.java
package com.cedarsoft.commons.javafx.properties;

import static org.assertj.core.api.Assertions.*;

import org.junit.jupiter.api.*;

/**
 */
public class KotlinFxPropertiesJavaTest {
  @Test
  void name() {
    Foo foo = new Foo();
    assertThat(foo.getName()).isEqualTo("");
    foo.setName("asdf");
    assertThat(foo.getName()).isEqualTo("asdf");
    assertThat(foo.getNameProperty().get()).isEqualTo("asdf");

    foo.getAge();
    foo.setAge(17);
    foo.getAgeProperty();
  }
}

File: commons/javafx/src/test/kotlin/it/neckar/open/javafx/properties/KotlinFxPropertiesJavaTest.kt
package it.neckar.open.javafx.properties

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

/**
 */
class KotlinFxPropertiesJavaTest {
  @Test
  fun name() {
    val foo = Foo()
    assertThat(foo.name).isEqualTo("")
    foo.name = "asdf"
    assertThat(foo.name).isEqualTo("asdf")
    assertThat(foo.nameProperty.get()).isEqualTo("asdf")
    foo.age
    foo.age = 17.0
    foo.ageProperty
  }
}
